### PR TITLE
Add documentation around how to start ManageIQ using Foreman

### PR DESCRIPTION
**Pull request by carbonin**
https://github.com/ManageIQ/manageiq/pull/15669 adds an example Procfile to ManageIQ.

This PR describes some of the moving parts.

### CODE REVIEW

**Comment by jrafanie** [comment]
Can you write a talk article introducing this?

**Comment by carbonin** [comment]
@jrafanie http://talk.manageiq.org/t/running-workers-using-foreman-in-development/2585

File: developer_setup/foreman.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## run_single_worker.rb
+
+The `run_single_worker.rb` script allows a single worker to be started without needing a server to monitor it. You simply provide the worker class and optionally a queue name and port as environment variables and the worker will start.
+
+There are additional options available which can be seen in the script's usage documentation.
+
+## Foreman
+
+[Foreman](https://ddollar.github.io/foreman/) is a simple process management tool which will start up a group of processes at the same time.
+It handles things like spinning up multiple instances of a partucular process and even will increment the built in PORT environment variable for each worker and worker type.
```
Comment (jrafanie) [inline developer_setup/foreman.md]: typo:  particular

File: developer_setup/foreman.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+The ManageIQ application can run as a collection of worker processes, without a centralized server. As a developer it may be easier to start the worker that you are currently working on, rather than starting the whole server and letting it launch the worker for you.
+
+## run_single_worker.rb
+
+The `run_single_worker.rb` script allows a single worker to be started without needing a server to monitor it. You simply provide the worker class and optionally a queue name and port as environment variables and the worker will start.
```
Comment (jrafanie) [inline developer_setup/foreman.md]: We probably want to point out the one major change is the starting ports of the web service and web socket workers.

Comment (carbonin) [inline developer_setup/foreman.md]: Added

File: developer_setup/foreman.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Foreman
+
+[Foreman](https://ddollar.github.io/foreman/) is a simple process management tool which will start up a group of processes at the same time.
+It handles things like spinning up multiple instances of a particular process and even will increment the built in PORT environment variable for each worker and worker type.
+
+### On ports
+Foreman's port number handling introduces one big change in that the port assignment is not the same as the default ports for our workers. Foreman increments the port starting number by 100 for each worker type and 1 for each worker instance.
+
+This means that the first worker listed in your Procfile will get port 3000, the second will get port 3100, etc.
+For our example Procfile, this means that the UI worker will get port 3000, the API worker will get 3100 and the websocket worker will get 3200. If you need a particular port for a worker, you can still set the environment variable manually in the Procfile.
```
Comment (jrafanie) [inline developer_setup/foreman.md]: 👍

**Pull request merged**